File: ios/MyFirstApp/MyFirstApp/ViewController.m
//
//  ViewController.m
//  MyFirstApp
//
//  Created by fujita taisuke on 2012/12/24.
//  Copyright (c) 2012年 9revolution9.com All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    // xy座標(0, 0)、幅100、高さ100のUIViewクラスのインスタンスを生成する
    UIView *view = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 100, 100)];
    // UIViewオブジェクトの背景色のデフォルト値は透明なため背景色を別の色に設定
    view.backgroundColor = [UIColor redColor];
    // viewプロパティにサブビューを追加する
    [self.view addSubview:view];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: ios_swift/SharedData/SharedData/ViewController.swift
//
//  ViewController.swift
//  SharedData
//
//  Created by taisuke fujita on 2014/06/29.
//  Copyright (c) 2014年 taisuke fujita. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
                            
    override func viewDidLoad() {
        super.viewDidLoad()
        let app = UIApplication.sharedApplication().delegate
                as AppDelegate
        app.sharedData["item1"] = "an item"
        
        let button = UIButton.buttonWithType(UIButtonType.System) as UIButton
        button.frame = CGRect(x: 0, y: 0, width: 50, height: 44)
        button.setTitle("開く", forState: UIControlState.Normal)
        button.addTarget(self,
            action: "respondToButtonClick:",
            forControlEvents: UIControlEvents.TouchUpInside)
        view.addSubview(button)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    func respondToButtonClick(sender: UIButton!) {
        let controller = ModalViewController()
        controller.view.backgroundColor = UIColor.whiteColor()
        self.presentViewController(controller,
            animated: true, completion: nil)
    }
}

